ansible/roles/omero-build/tasks/main.yml

```diff
@@ -35,6 +35,13 @@
     - python-pip
     - python-virtualenv
 
+- name: system packages | pip install packages
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - genshi
```

Comment:
For reasons I'm not entirely sure about, this role is also installing all the bio-formats dependencies as well.